commons/protocol/src/test/resource/saved-protocol/0.1/event/value_changed/int.json

```diff
@@ -1 +1 @@
-{"key":{"key":{"name":"name","manifest":{"erasureClass":"java.lang.String","typeArguments":[]}},"scope":{"project":{"build":"file:///test/project","name":"test"}}},"value":{"success":true,"serialized":42,"stringValue":"42"},"type":"ValueChanged"}
```

Comment:
This diff is really unhelpfully noisy for compiling notes for @warry on what will break the JS, but one thing I see is that we dropped `success:true` and now just use $type. Also of course we changed `type` to `$type`.